cmd/qgb/main.go

```diff
@@ -4,6 +4,11 @@ import (
 	"context"
 	"os"
 
+	"github.com/celestiaorg/celestia-app/x/qgb/client"
+	"github.com/celestiaorg/orchestrator-relayer/cmd/qgb/deploy"
+	"github.com/celestiaorg/orchestrator-relayer/cmd/qgb/orchestrator"
+	"github.com/celestiaorg/orchestrator-relayer/cmd/qgb/relayer"
+
```

Comment:
I think it's cool to leave them like this, personal opinion, because the imports from `orchestrator-relayer` repo are grouped together and other imports are separate. Also, the linter didn't complain.

But I'm fine with both :D